### ISSUE TITLE
pippo-bom

### ISSUE THREAD

**Issue by whyicantusemyemailasusername** [pippo-java/pippo#509]
It would be nice to have maven bom file like https://github.com/apache/logging-log4j2/blob/master/log4j-bom/pom.xml , to be able to specify pippo version only once like I do for log4j:

```xml
            <dependency>
                <groupId>org.apache.logging.log4j</groupId>
                <artifactId>log4j-bom</artifactId>
                <version>${log4j.version}</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
```

Now I have to specify the version for each pippo artefact which is not that convenient.

**Comment by decebals** [comment]
Can you supply a PR please?

**Comment by mhagnumdw** [comment]
Any progress here? I believe this will resolve my case below.

I thought about it now when I needed to get the version of Google Guice used by Pippo defined [here](https://github.com/pippo-java/pippo/blob/master/pippo-controller-parent/pippo-guice/pom.xml#L18).

**Comment by decebals** [comment]
> Any progress here? I believe this will resolve my case below.
> 
> I thought about it now when I needed to get the version of Google Guice used by Pippo defined [here](https://github.com/pippo-java/pippo/blob/master/pippo-controller-parent/pippo-guice/pom.xml#L18).

I use the same approach in my projects (for all major libraries or multi modules libraries, I specify the version using a property in `properties` section of the `pom.xml` file).

**Comment by decebals** [comment]
@mhagnumdw 

I feel that my answer is not what you expected :smile:. I added some text on [chat](https://gitter.im/decebals/pippo).

Regarding "BOM", I did not use it in my projects and I do not know what it brings valuable.

**Comment by mhagnumdw** [comment]
Hi @decebals , sorry, I could have already informed you why I need this.

> I thought about it now when I needed to get the version of Google Guice used by Pippo defined [here](https://github.com/pippo-java/pippo/blob/master/pippo-controller-parent/pippo-guice/pom.xml#L18).

I make use of `guice-persist`, `guice-assistedinject` and `guice-multibindings`, and I would like to keep their version synchronized with the version of `guice` brought by `pippo-guice`.

> Regarding "BOM", I did not use it in my projects and I do not know what it brings valuable.

_**What is Maven BOM?**_
_BOM stands for Bill Of Materials. A BOM is a special kind of POM that is used to control the versions of a project’s dependencies and provide a central place to define and update those versions._
_BOM provides the flexibility to add a dependency to our module without worrying about the version that we should depend on._

Ref: https://www.baeldung.com/spring-maven-bom#2-what-is-maven-bom
